device/bgp/bgpv4.yaml

```diff
@@ -1,21 +1,39 @@
 components:
   schemas:
-    Device.Bgpv4:
+    Bgp.V4Peer:
       x-status: under-review
       x-include: 
       - './bgp.yaml#/components/schemas/Device.Bgp'
       description: >-
         Container for emulated BGPv4 peers and routes.
       type: object
-      required: [local_address, dut_address]
+      required: [dut_address]
       properties:
-        local_address:
+        dut_address:
```

Comment:
shouldn't we call it `peer_address` ?

Reply:
IPv4 address of the BGP peer for the session. --> This is also an correct.